server/app.js

```diff
@@ -5,5 +5,10 @@ const app = express();
 
 app.use('/', express.static(path.join(__dirname, '../public')));
 app.use('/api/reviews', require('./routes/reviews.js'));
+app.use('/api/reviewsummary', require('./routes/reviewsummary.js'));
```

Comment:
Based on your Slack message, you have since refactored this api route, but I'm assuming this has been updated in a later commit.

Reply:
Correct. the next PR will have the change.